crates/db_views/src/post_view.rs

```diff
@@ -397,9 +397,11 @@ fn queries<'a>() -> Queries<
     if let Some(search_term) = &options.search_term {
       let searcher = fuzzy_search(search_term);
       query = query.filter(
-        post::name
+        (post::name
           .ilike(searcher.clone())
-          .or(post::body.ilike(searcher)),
+          .or(post::body.ilike(searcher)))
+        .and(post::removed.eq(false))
+        .and(post::deleted.eq(false)),
```

Comment:
Use `.filter(not(post::removed.or(post::deleted)))`

Reply:
Good call.

Reply:
Good ole DeMorgan's Law.